### ISSUE TITLE
Difficult to leave a space in which you are an admin

### ISSUE THREAD

**Issue by InfiniteWorldSoftware** [vector-im/element-web#16493]
![image](https://user-images.githubusercontent.com/66080865/108525675-5916d580-729e-11eb-8d4e-16f150b1efed.png)

### Description

I've noticed that it is still relatively difficult UI-wise to leave a space.
Currently, you have to go into Space Settings, and find the "Leave Space" button.

I would like there to be a "Leave Space" button at the bottom of the Space right-click menu similar to the room one shown below: 
![image](https://user-images.githubusercontent.com/66080865/108525968-9e3b0780-729e-11eb-81e0-5d5a4ebd1c12.png)

### Version information
Latest Spaces deployment

**Comment by t3chguy** [comment]
> I would like there to be a "Leave Space" button at the bottom of the Space right-click menu similar to the room one shown below:

There is, except not for admins.

Admins have to leave in Space Settings, non-Admins don't see Settings and they get a Leave interaction instead.

**Comment by InfiniteWorldSoftware** [comment]
Hm, okay. That's not exactly clear, and I have left rooms as admin before.
There is a lack of consistency here that is a bit confusing to me since I haven't used Spaces that much.

**Comment by t3chguy** [comment]
![image](https://user-images.githubusercontent.com/2403652/108526874-8f1d7f00-72c9-11eb-8aa5-e8f55347193b.png)

Its more to discourage admins leaving spaces without proper intent as it is quite likely an irreversible action which may leave a Space completely admin-less and unremovable.

**Comment by InfiniteWorldSoftware** [comment]
Maybe the warning dialogue could be different for admins? 
For example, it could say, "Warning: You are an admin of this space. do you really want to leave?"

**Comment by InfiniteWorldSoftware** [comment]
A similar problem exists for rooms- no one can administrate a room if all the admins leave.

**Comment by t3chguy** [comment]
We identified an Admin leaving a space to be a very rare action so it didn't need to be exposed on the quick actions menu.

**Comment by t3chguy** [comment]
> A similar problem exists for rooms- no one can administrate a room if all the admins leave.

Yup, we are experimenting with this show-relevant-quick-actions approach with Spaces, it'll likely not be iterated without proper user testing

**Comment by InfiniteWorldSoftware** [comment]
This is my preference, but I defintely prefer warning dialogues such as those that pop up when trying to leave a room rather than hiding an option completely. I had incorrectly assumed that it was impossible to leave a space until digging a little deeped in settings.

**Comment by t3chguy** [comment]
FTR Room Settings also has Leave Room option, only change is that the Quick Actions are not exhaustive and are scoped to be more relevant to the actions you are likely to take so that it doesn't turn into a list of ~10 options.

![image](https://user-images.githubusercontent.com/2403652/108530505-947cc880-72cd-11eb-98d9-e82e737d4eab.png)

Do you expect the `Invite people` option to be available even if you don't have permission which just errors for consistency?

![image](https://user-images.githubusercontent.com/2403652/108530689-c42bd080-72cd-11eb-8a55-e703bc333635.png)

**Comment by InfiniteWorldSoftware** [comment]
No, since there is an "Invite to this Space" button in the right panel.

**Comment by kittykat** [comment]
Closing as this is not fixed:
![Screenshot from 2021-09-17 11-28-25](https://user-images.githubusercontent.com/51663/133768399-bde6c4f6-ad08-486e-ba2a-7f4a3bc1adcd.png)

**Comment by robintown** [comment]
This is not fixed, spaces in which you are an admin still hide the 'leave' option from menus
